src/components/Popups/ReplyField/__tests__/PopupList-test.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { shallow, ShallowWrapper } from 'enzyme';
+import PopupBase from '../../PopupBase';
+import PopupList, { Props } from '../PopupList';
+
+describe('components/Popups/ReplyField/PopupList', () => {
```

Comment:
What's the test coverage on this component?

Reply:
> What's the test coverage on this component?

100% 😂